common/laws/00_land_reform.txt

```diff
@@ -112,6 +116,10 @@ law_commercialized_agriculture = {
 	progressiveness = 0
 	modifier = {
 		tax_per_capita_add = 0.05
+		interest_group_ig_industrialists_pop_attraction_mult = 0.1
+		country_trade_route_competitiveness_mult = 0.05
+		country_trade_route_quantity_mult = 0.1
+		country_loan_interest_rate_mult = -0.05
```

Comment:
I would recommend not to mess with the loans interest rate at secondary places like this. There just is a point where debt becomes free and that is hard enough to manage as it is with its various techs and laws.

Reply:
Can't we institute a minimum? this is not for the flat loan value but a multiplier of the base rate